doctr/datasets/vocabs.py

```diff
@@ -15,6 +15,11 @@
     'punctuation': string.punctuation,
     'currency': '£€¥¢฿',
     'ancient_greek': 'αβγδεζηθικλμνξοπρστυφχψωΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ',
+    'arabic_letters': 'ءآأؤإئابةتثجحخدذرزسشصضطظعغـ‌‍‎‏فقكلمنهوىي',
+    'arabic_farsi_letters': 'ٯٰپچژڜڢڤڥڧڨکگی',
+    'hindi_numbers': '٠١٢٣٤٥٦٧٨٩',
+    'arabic_diacritics': 'ًٌٍَُِّْ',
+    'arabic_unique_punctuation': '؟؛«»—'
```

Comment:
while you're at it, could you rename it `arabic_punctutation` please?

Reply:
`arabic_punctuation`